src/app/Ordino.ts

```diff
@@ -33,39 +35,40 @@ export class Ordino extends ATDPApplication<OrdinoApp> {
 
     const app: OrdinoApp = new modules[0].OrdinoApp(graph, manager, main);
 
+    const modePromise = new Promise<'start'|'overlay'>((resolve, reject) => {
+      this.modeResolver = resolve;
+    });
+
     const startMenuElement = main.ownerDocument.createElement('div');
     main.parentElement.append(startMenuElement); // append element before ReactDOM.render()
-    const renderStartMenu = () => ReactDOM.render(React.createElement(modules[1].StartMenuComponent, {headerMainMenu: this.header.mainMenu, manager, graph}), startMenuElement);
+    const renderStartMenu = () => ReactDOM.render(React.createElement(modules[1].StartMenuComponent, {headerMainMenu: this.header.mainMenu, manager, graph, modePromise}), startMenuElement);
```

Comment:
@puehringer I'm not sure if there is a better way than to pass the promise to the start menu. I will explain it to you in a meeting.